src/Tizen.NUI/src/public/Utility/Capture.cs

```diff
@@ -65,32 +67,75 @@ internal Capture(IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemoryOwn)
         }
 
         /// <summary>
-        /// Dispose
+        /// Top left position of capturing region
+        /// Default is Position(0, 0)
         /// </summary>
-        /// <param name="type"></param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected override void Dispose(DisposeTypes type)
+        /// <since_tizen> 9 </since_tizen>
+        public Position Position { get; set; } = new Position(0, 0, 0);
+
+        /// <summary>
+        /// Size of capturing region
+        /// Size value for each direction is required larger than zero.
+        /// Default is window size.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public Size Size { get; set; } = new Size(0, 0, 0);
+
+        /// <summary>
+        /// image file path to be saved as a file.
+        /// If path is empty string, the captured result is not be saved as a file.
```

Comment:
'is not be saved' this sentence uses duplicated verbs,, :)

```suggestion
        /// If path is empty string, the captured result is not saved as a file.
```

Reply:
Good thank you.